### DISCUSSION TITLE
How to know what Android API is supported for .netX.0-android?

### DISCUSSION THREAD

**Discussion by jeanplevesque** [unoplatform/uno#11423] [Q&A] [answered]
I'm looking at the frameworks of the Uno.WinUI package and I have a few questions.
- For MonoAndroid it's pretty clear that Android 12 and 13 are supported.
- For .net-android however, it shows 30.0 which I assume is the API level, which in this case would mean Android 11.

1. **Does that mean that for .net-android, Android 12 and 13 are not supported for this version of the package?**
2. In the Xamarin era, we used to update our multi-targeted libraries to add support for the newer Android version each year (by adding `monoandroid12.0`, then `monoandroid13.0`, etc.).
   **How does that translate for .net-android?**

![image](https://user-images.githubusercontent.com/39710855/220685777-82e77221-4929-4683-a358-5844183a8995.png)

**Comment by jeromelaban** [comment, marked as answer]
The same way previous versions of Uno were forward compatible more recent versions of Android, this works the same way here. the current build of Uno works with `android30.0` and later. It is likely that in the future, major .NET versions will target an updated version of Android.

Is this question linked to a specific issue you've found?

**Reply by jeanplevesque** [reply]
> Is this question linked to a specific issue you've found?

We want to update some packages to target Android 13, but from what I understand now, they probably are already compatible and the only thing that needs updating is the head project?

**Reply by Iguins** [reply]
If we want to use a specific api in android 13
I think we will need a version that targets net6.0-android33.0
https://github.com/xamarin/xamarin-android/pull/6896

**Reply by jeromelaban** [reply]
> We want to update some packages to target Android 13, but from what I understand now, they probably are already compatible and the only thing that needs updating is the head project?

That's correct, or in some of your dependencies. If that becomes a problem in the future, we'll introduce separate `net8.0-android33.0` and `net8.0-android34.0` where that makes sense.
